modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -124,6 +124,17 @@ covers the following regions.
 
 |===
 
+https://anypoint.mulesoft.com[Anypoint Platform] for the MuleSoft Government Cloud
```

Comment:
remove the "the"? I don't think we use articles with product names.

Reply:
Removed the "teh". Left the "." to be consistent with the other sections and minimize changes to your file.